CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# Contributing to bible memorization
+
+- [Contributing to bible memorization](#contributing-to-bible-memorization)
+  - [Getting Started](#getting-started)
+  - [Pulls Requests](#pulls-requests)
+  - [Submitting a Pull Request](#submitting-a-pull-request)
+
+## Getting Started
+
+Start by **starring** the repo or [forking](https://github.com/garretlin/bible-memorization/fork) it.
+
+## Pulls Requests
+
+PRs can be [opened](#submitting-a-pull-request) on our [GitHub Repository][GitHub] and used for the following purposes adhering to their respective guidelines
+
+When deciding if we merge in a pull request, we will require:
+
+* at least 1 review
+* good quality code
+    * no spelling mistakes
+    * good grammer
+    * clearly stated changes
+
+## Submitting a Pull Request
+
+Follow these steps and guidelines when starting with a PR:
+* Make your changes in a new local git branch:
+     ```shell
+     git checkout -b BRANCH_NAME origin/master
+     ```
+* Use one branch per fix / feature, and each branch should have the prefix `feature` or `bugfix` followed by a backslash
+* Commit your changes
+    * Please provide a git message that explains what you've done
+    * Make sure your commit messages follow the [conventional guidelines](https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53#file-commit-message-guidelines-md)
+    * Commit to the forked repository
+
+        Example :pencil2: :
+        ```
+        $ git commit -am 'Fixing sum ting wong'
+        ```
+    * Push to the branch in the forked repository
+        
+        Example :pencil2: :
+        ```
+        $ git push origin BRANCH_NAME
+        ```
+
+After making sure that all above guidelines have been followed completely,
```

Comment:
Should we also try for a cleanup requirement after the PR to
- delete the branch after merging (not sure if that's a setting that I need to put on the repo)
- squash commits into a single commit (also not sure if there's something we can set as a setting or if there's some basic CI/CD stuff we can do to check that)

Reply:
both are done by GitHub setting
[deleting](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-the-automatic-deletion-of-branches)
[squashing](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-squashing-for-pull-requests)